concourse/pipelines/linux-image-build.yaml

```diff
@@ -1070,6 +1070,22 @@ jobs:
       wf: "debian/debian_11.publish.json"
       environment: "test"
 
+# Image test
+# centos
+- name: image-test-centos-7
```

Comment:
does this need to be a job? or can it be a task?

Reply:
It could be a task.

Reply:
Discuss offline. Going to make the test as a task.